packages/cli/commands/create.js

```diff
@@ -111,6 +112,10 @@ exports.builder = yargs => {
     describe: i18n(`${i18nKey}.positionals.dest.describe`),
     type: 'string',
   });
+  yargs.option('internal', {
+    describe: 'Internal HubSpot version of creation command',
+    type: 'boolean',
+  });
```

Comment:
Perhaps we should make this a hidden option? I think you can just add `hidden: true` to the props here but I don't remember the yargs API that well.

Reply:
Yep - just checked that out in docs -- added hidden property to that option